specs/endpoints/food_delete.spec.js

```diff
@@ -0,0 +1,101 @@
+var shell = require('shelljs');
+var request = require("supertest");
+var app = require('../../app');
+var Food = require('../../models').Food;
+
+describe('Food delete API', () => {
+  beforeAll(() => {
+    shell.exec('npx sequelize db:create')
+  });
+  beforeEach(() => {
+    shell.exec('npx sequelize db:migrate')
+    shell.exec('npx sequelize db:seed:all')
+  });
+  afterEach(() => {
+    shell.exec('npx sequelize db:seed:undo:all')
+  });
+
+  describe('', () => {
+    test('it should delete a food successfully', () => {
+      // Below is an alternative way to create a food. Although, by utilizing this way,
+      // I am receiving the error that food cannot be created without a name, I don't understand
+      // why the given name of Cheese Puffs is not being utilized.
+
+      // const id = 83
+      // Food.create({
+      //   where: {
+      //     id: id,
```

Comment:
you don't need to supply an Id when creating a food

Reply:
The intent of this was to create a food with an id that is consistent every time the test is run. I tried this with and without the id, and it still was not creating the food item with the name.